tests/foreman/virtwho/cli/test_nutanix.py

```diff
@@ -171,5 +170,85 @@ def test_positive_hypervisor_id_option(
+            The prism_central has been set in /etc/virt-who.d/vir-who.conf file
+
+        :CaseLevel: Integration
+
+        :CaseImportance: High
+        """
+        form_data['prism-flavor'] = "central"
+        virtwho_config = target_sat.cli.VirtWhoConfig.create(form_data)['general-information']
+        assert virtwho_config['status'] == 'No Report Yet'
+        if deploy_type == "id":
+            command = get_configure_command(virtwho_config['id'], default_org.name)
+            hypervisor_name, guest_name = deploy_configure_by_command(
+                command, form_data['hypervisor-type'], debug=True, org=default_org.label
+            )
+        elif deploy_type == "script":
+            script = target_sat.cli.VirtWhoConfig.fetch(
+                {'id': virtwho_config['id']}, output_format='base'
+            )
+            hypervisor_name, guest_name = deploy_configure_by_script(
+                script, form_data['hypervisor-type'], debug=True, org=default_org.label
+            )
+        # Check the option "prism_central=true" should be set in etc/virt-who.d/virt-who.conf
+        config_file = get_configure_file(virtwho_config['id'])
+        assert get_configure_option("prism_central", config_file) == 'true'
+        virt_who_instance = target_sat.cli.VirtWhoConfig.info({'id': virtwho_config['id']})[
+            'general-information'
+        ]['status']
+        assert virt_who_instance == 'OK'
+        hosts = [
+            (hypervisor_name, f'product_id={settings.virtwho.sku.vdc_physical} and type=NORMAL'),
+            (guest_name, f'product_id={settings.virtwho.sku.vdc_physical} and type=STACK_DERIVED'),
+        ]
+        for hostname, sku in hosts:
+            host = target_sat.cli.Host.list({'search': hostname})[0]
+            subscriptions = target_sat.cli.Subscription.list(
+                {'organization': default_org.name, 'search': sku}
+            )
+            vdc_id = subscriptions[0]['id']
+            if 'type=STACK_DERIVED' in sku:
+                for item in subscriptions:
+                    if hypervisor_name.lower() in item['type']:
+                        vdc_id = item['id']
+                        break
+            result = target_sat.cli.Host.subscription_attach(
+                {'host-id': host['id'], 'subscription-id': vdc_id}
+            )
+            assert result.strip() == 'Subscription attached to the host successfully.'
+
+    @pytest.mark.tier2
+    def test_positive_prism_central_prism_central_option(
+        self, default_org, form_data, virtwho_config, target_sat
+    ):
+        """Verify prism_central option by hammer virt-who-config update"
+
+        :id: 701e0390-2bfb-404a-a9bd-fa0fb5ecfdf8
+
+        :expectedresults: prism_central option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        value = 'central'
+        target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'prism-flavor': value})
```

Comment:
Here, it would be nice to assert the RC of the update command, non-blocking/optional

Reply:
@Gauravtalreja1 Do you mean add assert for virt-who config update?
-        target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'prism-flavor': value})
+        result = target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'prism-flavor': value})
+        assert result[0]['message'] == f"Virt Who configuration [{virtwho_config['name']}] updated"

Reply:
> Here, it would be nice to assert the RC of the update command, non-blocking/optional

@Gauravtalreja1 Do you mean add assert for virt-who config update as follow?
result = target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'prism-flavor': value})
assert result[0]['message'] == f"Virt Who configuration [{virtwho_config['name']}] updated"

Reply:
> > Here, it would be nice to assert the RC of the update command, non-blocking/optional
> 
> @Gauravtalreja1 Do you mean add assert for virt-who config update as follow? result = target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'prism-flavor': value}) assert result[0]['message'] == f"Virt Who configuration [{virtwho_config['name']}] updated"

@Gauravtalreja1 Have already added assert for this part, please review it ,many thanks

Reply:
@yanpliu Thanks for the update